.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - helm
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
```

Comment:
Is this used to make commit?
I can imagine this creates a unverified commit or verification problem for the created commits if we try to make them in the name of a person who initiated the release.

I would recommend doing the same thing we do in the collector and agent:

- Author commits in the name of @appsignalreleasebot 
- Sign the commits using its signing key
- Then, if possible, add the GitHub actor in the commit message and/or tag.

Here's how it's done in the collector: https://github.com/appsignal/appsignal-collector/blob/1f6959017907f87acb3b4bf64e77a4239f850219/.github/workflows/publish_release.yml#L39-L51

You'll need to have the secrets for this set up, which I can do.